backend/models/transaction.model.js

```diff
@@ -5,28 +5,26 @@ module.exports = (sequelize, Sequelize) => {
             primaryKey: true
         },
         from_id: {
-            type: Sequelize.INTEGER
+            type: Sequelize.INTEGER,
+            primaryKey: true,
```

Comment:
I think request ID should be enough for primary key, than using from_ID and to_ID. request ID should be good